cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1025,16 +1039,16 @@ def _add_head_node(self):
 
         cfn_init = {
             "configSets": {
+                "deployFiles": ["deployConfigFiles"],
                 "default": [
-                    "deployConfigFiles",
                     "cfnHupConfig",
                     "chefPrepEnv",
                     "shellRunPreInstall",
                     "chefConfig",
                     "shellRunPostInstall",
                     "chefFinalize",
                 ],
-                "update": ["deployConfigFiles", "chefUpdate"],
+                "update": ["deployConfigFiles", "chefUpdate", "sendSignal"],
```

Comment:
what if chefUpdate fails?in that case we need to signal a failure.

Reply:
good catch, added failure siganl, but in worst case, it will be timeout in 30 minutes.